custom_command.py

```diff
@@ -0,0 +1,56 @@
+""" This file aims to demonstrate how to write custom commands in OpenWPM
+
+Steps to have a custom command run as part of a CommandSequence
+
+1. Create a class that derives from BaseCommand
+2. Implement the execute method
+3. Append it to the CommandSequence
+4. Execute the CommandSequence
+
+The following parameters get passed to the execute function:
+1. webdriver: WebDriver is a Selenium class used to control
+   browser. You can simulate arbitrary interactions and extract almost all browser state
+   with the tools that Selenium gives you
+2. browser_params: Configuration that might differ per browser
```

Comment:
Move this into either the `BaseCommand.execute()` and `BrowserParams` and `ManagerParams`